WordPress/Classes/ViewRelated/Jetpack/Branding/Menu Card/JetpackBrandingMenuCardPresenter.swift

```diff
@@ -190,8 +198,9 @@ private extension JetpackBrandingMenuCardPresenter {
         static let phaseFourTitle = NSLocalizedString("jetpack.menuCard.phaseFour.title",
                                                            value: "Switch to Jetpack",
                                                            comment: "Title of a button prompting users to switch to the Jetpack app.")
-        static let newUsersPhaseTitle = NSLocalizedString("jetpack.menuCard.newUsers.title",
-                                                           value: "Unlock your site’s full potential. Get stats, notifications and more with Jetpack.",
-                                                           comment: "Description inside a menu card prompting users to switch to the Jetpack app.")
+        static let newUsersPhaseDescription = NSLocalizedString("jetpack.menuCard.newUsers.title",
+                                                                value: "Unlock your site’s full potential. Get stats, notifications and more with Jetpack.",
```

Comment:
Missing a mention about the Reader?
```suggestion
                                                                value: "Unlock your site’s full potential. Get Stats, Reader, Notifications and more with Jetpack.",
```